app/jobs/export_hathi_sip_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class ExportHathiSipJob < ApplicationJob
+  def perform(resource_id, destination = Rails.root.join('tmp'))
+    logger.info "Exporting #{resource_id} to Hathi SIP in '#{destination}'"
```

Comment:
Why don't you pass the logger in so that the rake task can set it to `STDOUT`. See that code example that I will look up.